crates/ruff_linter/resources/test/fixtures/pyflakes/F722.py

```diff
@@ -11,3 +11,30 @@ def g() -> "///":
 
 
 X: """List[int]"""'☃' = []
+
+y: """
+
+   int |
+   str
+"""
+
+z: """(
+
+    int |
+    str
+)
+"""
+
+# single quotes are not implicitly parenthesized
+invalid: "\n int"
+
+
+invalid2: """
+		int |
+str)
+"""
+
+invalid3: """
+		int) |
+str
+"""
```

Comment:
Can we add some test cases for unclosed parenthesis similar to how these? It would also be useful to provide test cases with multiple parentheses.